src/scripts/main.js

```diff
@@ -1,3 +1,168 @@
 'use strict';
+// import { pushNotification } from './notification.js';
 
-// write code here
+// Sort table
+
+const sortButtons = document.querySelectorAll('thead tr th');
+const tbody = document.querySelector('tbody');
+const rows = Array.from(document.querySelectorAll('tbody tr'));
+
+const sortState = new Array(sortButtons.length).fill(false);
+
+sortButtons.forEach((button, index) => {
+  button.addEventListener('click', () => {
+    const isDescending = sortState[index];
+
+    const sortedRows = rows.sort((rowA, rowB) => {
```

Comment:
all added rows disappear after sorting
You should get all current rows inside sort function to fix it